src/WCSimGenerator_Radioactivity.cc

```diff
@@ -271,1170 +142,525 @@ void WCSimGenerator_Radioactivity::Configuration(G4int iScenario, G4double dLife
 	if ( dLifeTime > 0 )
 		fRnLambda = log(2) / (dLifeTime);
 	this->SetScenario(iScenario);
+	G4cout << " ========================================================================== " << G4endl;
 	
 	G4cout << " Livetime: " << log(2) / fRnLambda << " sec " <<  G4endl;
```

Comment:
```suggestion
	G4cout << " Mean lifetime: " << log(2) / fRnLambda << " sec " <<  G4endl;
```
I think this is meant to be mean lifetime, rather than livetime (which puts me in mind of "what time period is being simulated?")

Reply:
Yeah, I am sorry, I very often misspell "lifetime"